anax-core/src/main/java/org/anax/framework/controllers/AppiumDriverWebController.java

```diff
@@ -559,6 +564,17 @@ public String getPageSource() {
         return null;
     }
 
+    @Override
```

Comment:
Instead of adding `restart()` as implementation to all drivers, consider adding a `default` on the interface (that returns false) and only implement in `WebController`s that can possibly do it.